doc/README.md

```diff
@@ -124,5 +124,5 @@ The produced documents are stored in the described directories above,
 you can install them just by copying the contents to the desired
 directory.
 
-In the future, the `coq-doc` and `coq-stdlib` opam packages will
+In the future, the `coq-doc` and `rocq-init` opam packages will
```

Comment:
Will it change to `rocq-doc`?

Reply:
All the renaming of existing packages will happen in other PRs (independent of this one).